src/ui/common/src/components/workflows/edges/AqueductBezier.tsx

```diff
@@ -13,14 +13,16 @@ export const AqueductBezier: React.FC<EdgeProps> = ({
   targetPosition,
   style = {},
 }) => {
-  const edgePath = getBezierPath({
+  // path, labelX, labelY, offsetX, offsetY
+  const edgePathData = getBezierPath({
     sourceX,
     sourceY,
     sourcePosition,
     targetX,
     targetY,
     targetPosition,
   });
+  const edgePath = edgePathData[0];
```

Comment:
What is edgePathData[0], or rather what items do we get back in the edgePathData array?

Reply:
The output of getBezierPath are the following items: path, labelX, labelY, offsetX, offsetY.

It seems they updated the return of the function which is why we encountered the issue